pkg/controller/machineset/controller.go

```diff
@@ -158,41 +210,32 @@ func (c *MachineSetControllerImpl) createMachine(machineSet *v1alpha1.MachineSet
 	return machine
 }
 
-// getMachines returns a list of machines that match on machineSet.Spec.Selector
-func (c *MachineSetControllerImpl) getMachines(machineSet *v1alpha1.MachineSet) ([]*v1alpha1.Machine, error) {
-	selector, err := metav1.LabelSelectorAsSelector(&machineSet.Spec.Selector)
-	if err != nil {
-		return nil, err
-	}
-	filteredMachines, err := c.machineLister.List(selector)
-	if err != nil {
-		return nil, err
-	}
-	return filteredMachines, err
-}
-
-func shouldAdopt(machineSet *v1alpha1.MachineSet, machine *v1alpha1.Machine) bool {
-	// Do nothing if the machine is being deleted.
-	if !machine.ObjectMeta.DeletionTimestamp.IsZero() {
+// filterMachine returns true if it is a machine to be considered for the machineset. False otherwise.
+func (c *MachineSetControllerImpl) filterMachine(machineSet *v1alpha1.MachineSet, machine *v1alpha1.Machine) bool {
```

Comment:
(nit)
Without reading the comment, I would have thought that filterMachine would return true if the machine should be filtered instead of the opposite.

Reply:
inverted it and updated comment.